services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerMultiClusterConfig.java

```diff
@@ -79,6 +80,10 @@ public boolean isParent() {
     return getCommonConfig().isParent();
   }
 
+  public ConcurrentPushDetectionStrategy getConcurrentPushDetectionStrategy() {
```

Comment:
Can we make this per cluster config so that we can control the rollout granularity?